pkg/proxy/winkernel/hcnutils_test.go

```diff
@@ -42,28 +42,65 @@ const (
 	protocol          = 6
 	internalPort      = 80
 	externalPort      = 32440
+	policyConst       = "50"
+	isolationID       = 5000
 )
 
+func assertHCNDiff(x, y interface{}) string {
+	// We can ignore some fields since they are known only at creation time
+	var nx, ny interface{}
+	switch x.(type) {
+	case hcn.HostComputeEndpoint:
+		nx := x.(hcn.HostComputeEndpoint)
```

Comment:
This looks wrong. ``:=`` is meant for declaration and asignment, which means that there's a new ``nx`` and ``ny`` being declared here, which are not the same as the ``nx`` and ``ny`` above. This means that the diff below will always be empty, which will result in false positives.

Same issue in the other cases too.